frontend/templates/layouts/sitemap.xml

```diff
@@ -1,11 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
   {% for doc in docs|sort(attribute='url') if not doc.sitemap.enabled is sameas false %}
+  {% if not '/documentation/examples/previews' in doc.pod_path %}
```

Comment:
Can we not use the hard-coded path for this?

Reply:
That's just as we are doing it in structured-data.j2: https://github.com/ampproject/amp.dev/blob/5a7b0f89a95078499088006d1df80aedd7e57852/frontend/templates/views/partials/structured-data.j2#L29

Alternatively we could check for the template by using `doc.view == '/views/examples/preview.j2'`. What do you like better?

Reply:
Yup, I don't like it there either :-). Can we instead set a flag in the front matter (e.g. in the samples builder)?

Reply:
Sure!

Reply:
@matthiasrohmer This can then be used to hide the tests from the sitemap as well https://github.com/ampproject/amp.dev/issues/2802